utility/styles.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const insertImport = require('./module').insertImport;
+
+const defaultStyles = [
```

Comment:
I'm pretty sure that the default styles should be defined in `application.react.js` file. A utility should be abstract. And maybe, in this case, this utility is not needed at all.